rmw_zenoh_cpp/config/DEFAULT_RMW_ZENOH_SESSION_CONFIG.json5

```diff
@@ -24,7 +24,7 @@
   /// peers, or client can use to establish a zenoh session.
   listen: {
     endpoints: [
-      // "tcp/localhost:7447",
+      "tcp/localhost:0",
```

Comment:
Could you add the explanation in the PR description as a comment above this line?